raiden/api/rest.py

```diff
@@ -427,7 +413,25 @@ def serve_forever(self, host='127.0.0.1', port=5001):
         return self.get()  # block here
 
     def start(self, host='127.0.0.1', port=5001):
+        # WSGI expects a stdlib logger, with structlog there's conflict of
+        # method names rest unhandled exception will be re-raised here:
+        wsgi_log = logging.getLogger(__name__ + '.pywsgi')
+        wsgiserver = WSGIServer(
+            (host, port),
+            self.flask_app,
+            log=wsgi_log,
+            error_log=wsgi_log,
+        )
+
+        try:
+            wsgiserver.init_socket()
+        except socket.error as e:
+            if e.errno == errno.EADDRINUSE:
+                raise APIServerPortInUseError()
+            raise
+
         self.kwargs = {'host': host, 'port': port}
```

Comment:
`args` and `kwargs` is passed to `_run`. If you remove the parameters from `_run`, there's no need to set it